connexion/apis/aiohttp_api.py

```diff
@@ -261,21 +262,44 @@ def _add_operation_internal(self, method, path, operation):
 
     @classmethod
     @asyncio.coroutine
-    def get_request(cls, req):
+    def get_request(cls, req: web.Request):
         """Convert aiohttp request to connexion
 
         :param req: instance of aiohttp.web.Request
         :return: connexion request instance
         :rtype: ConnexionRequest
         """
         url = str(req.url)
-        logger.debug('Getting data and status code',
-                     extra={'has_body': req.has_body, 'url': url})
+        content_type: str = req.content_type
+
+        logger.debug(
+            'Getting data and status code',
+            extra={
+                # has_body | can_raed_body report if
+                # body has been read or not
+                # body_exists refers to underlying stream of data
+                'has_body': req.body_exists,
+                'can_ready_body': req.can_read_body,
+                'content_type': content_type,
+                'url': url,
+            },
+        )
 
         query = parse_qs(req.rel_url.query_string)
         headers = req.headers
         body = None
-        if req.body_exists:
+
+        if content_type in http_facts.FORM_CONTENT_TYPES:
+            logger.debug('Reading multipart data from request')
+            data = yield from req.post()
+
+            files = {k: v for k, v in data.items() if isinstance(v, web.FileField)}
```

Comment:
__(nit)__ Having one loop will save some milliseconds for some high-load connexion service